test/index.html

```diff
@@ -370,6 +369,9 @@ <h2 id="qunit-userAgent"></h2>
 		<input id="radiocheckbox-1-2" autocomplete="off" type="checkbox" name="radiocheckbox-1" required="required">
 		<input id="radiocheckbox-1-3" autocomplete="off" type="checkbox" name="radiocheckbox-1" required="required">
 	</form>
+	<form id="validatorDestroy" autocomplete="off">
```

Comment:
Does this really need a new element? Looks like you could just reuse any other form with a require input.